### ISSUE TITLE
GripList and GripClean error

### ISSUE THREAD

**Issue by VincentCordobes** [PratikBhusal/vim-grip#3]
Commands  _GripList_, _GripClean_ throws errors:

```
Error detected while processing function <SNR>90_clean[2]..<lambda>5:                 
line    1:                                                                            
E474: Invalid argument                                                                
E691: Can only compare List with List
```
I am using neovim _v0.5.0-60158df_ on macos.

_Note that _GripStart_ works like a charm! 🙂_

**Comment by PratikBhusal** [comment]
Could you tell me the steps you took to achieve this?

**Comment by VincentCordobes** [comment]
```
vim README.md
:GripStart
:GripClean
```

**Comment by PratikBhusal** [comment]
I think I have found the issue. Expect a bug-fix within the next 24 hours.

**Comment by PratikBhusal** [comment]
@VincentCordobes I have fixed the bug with commit cf1cc5825edee11fb4a36371fe463d765201d679. For now, I will close this issue. Open it again if the problem still exists.

**Comment by VincentCordobes** [comment]
It's fixed, thank you 🙂
